compact/far_forward/ZDC.xml

```diff
@@ -25,47 +24,52 @@
       Zero Degree Calorimeter - ECAL
       -------------------------------
     </comment>
-    <constant name="ZDC_1stSilicon_z_pos"        value="ZDC_z_pos"/>
-    <constant name="ZDC_1stSilicon_x_pos"        value="ZDC_1stSilicon_z_pos * ionCrossingAngle"/>
+    <constant name="ZDC_1stSilicon_r_pos"        value="ZDC_r_pos+0.5*cm"/>
+    <constant name="ZDC_1stSilicon_z_pos"        value="ZDC_1stSilicon_r_pos * cos(ionCrossingAngle)"/>
+    <constant name="ZDC_1stSilicon_x_pos"        value=" -1. * ZDC_1stSilicon_r_pos * sin(ionCrossingAngle)"/>
```

Comment:
I'm surprised we need an explicit `-1` here, since I thought the so-called "big sign flip" made sure that `ionCrossingAngle` contained the correct sign to be able to treat this without additional factors of `-1` in the far forward region. @ajentsch @sly2j

Reply:
Thanks for pointing this out and sorry that I messed up. I will upload a new xml file today.